Automattic/fresh-data
# Update build for @babel/runtime

This library was failing to work with `woocommerce-admin` because of the `debug` dependency and how it
used `@babel/runtime` when the rest of the repo was using
`@babel/runtime-corejs2`. This attempts to solve that problem by
consolidating on `@babel/runtime` instead.

To Test:
1. `npm install`, `npm run bootstrap`, `npm run build`
2. `cd packages/framework`, `npm link`, `npm run build`
3. From `woocommerce-admin` build, `npm install`, `npm link @fresh-data/framework`, `npm start`
4. Open analytics in wp-admin and ensure reports come up and work.
